src/main/java/agents/leeroy/AttackActionSupplier.java

```diff
@@ -0,0 +1,52 @@
+package agents.leeroy;
+
+import at.ac.tuwien.ifs.sge.game.risk.board.Risk;
+import at.ac.tuwien.ifs.sge.game.risk.board.RiskAction;
+
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class AttackActionSupplier {
+
+    private static final double RISK_THRESHOLD = 0.5;
+
+    public static Set<RiskAction> createActions(Risk risk) {
+        final Set<Integer> sourceTerritoryIds = risk
+                .getBoard()
+                .getTerritoriesOccupiedByPlayer(risk.getCurrentPlayer());
+        var attackActions = sourceTerritoryIds
```

Comment:
Ich würde `sourceTerritoryIds` nicht anlegen und einfach direkt von :17 weiter `.stream` aufmachen

Reply:
ok - in this case I did it for readability, i.e. source/target distinction